### ISSUE TITLE
Input data can be different because of PIL/image

### ISSUE THREAD

**Issue by Cadene** [clcarwin/convert_torch_to_pytorch#9]
Hello there,

I ported resnet152 from torch7 to pytorch and got a lower accuracy (-0.4%) on the valset of imagenet.

The error seems to come from how pytorch load images (by default) using PIL. In fact, resnet152 from torch7 has been trained using the package `image` (`image.load(path)`) and both gives different tensors.

I made a repo to highlight the difference: https://github.com/Cadene/test_pil

Do you have any idea how to reduce the gap ?

**Comment by clcarwin** [comment]
You can try to fine-tune the converted model with a very small learning rate. The gap may be reduced in a few epochs.
